bindings/python/src/OpenSpaceToolkitAstrodynamicsPy/Flight/Profile.cpp

```diff
@@ -138,7 +143,19 @@ inline void OpenSpaceToolkitAstrodynamicsPy_Flight_Profile(pybind11::module& aMo
             "target_velocity",
             &Profile::TrajectoryTarget::TargetVelocity,
             R"doc(
-                Create a target, which produces a vector pointing along the target velocity.
+                Create a target, which produces a vector pointing along the scan direction.
+                When choosing this as a clocking target, the resulting profile will not be yaw compensated.
+            )doc",
+            arg("trajectory"),
+            arg("axis"),
+            arg("anti_direction") = false
+        )
+        .def_static(
+            "target_sliding_ground_velocity",
+            &Profile::TrajectoryTarget::TargetSlidingGroundVelocity,
+            R"doc(
+                Create a target, which produces a vector pointing along the ground velocity vector (aka the scan direction of the point sliding across the ground).
+                When choosing this as a clocking target, the resulting profile will be yaw compensated.
```

Comment:
```suggestion
                When choosing this as a clocking target, the resulting profile will compensate for the rotation of the referenced celestial body for the clocking axis.
```
It's not always yaw, depends on the axis selected and the frame definitions.